source/includes/_deployments.md

```diff
@@ -212,6 +214,11 @@ Property | Type | Description
 `inputs` | object | The dictionary containing key value pairs which represents the deployment inputs.
 `private_resource` | boolean | Optional parameter, if set to True the uploaded resource will only be accessible by its creator. Otherwise, the resource is accessible by all users that belong to the same tenant (default: False).
 `skip_plugins_validation` | boolean | Optional parameter, determines whether to validate if the required deployment plugins exist on the manager (default: False).
+`availability` | string | Optional parameter, defines who can access the deployment (default: tenant).
+
+Valid availability values are:
+* `private` - The resource is available to the owner (the creating user).
+* `tenant` - The default availability, the resource is available to all users in the current tenant.
```

Comment:
As above -> tenant - The resource is available to all users in the current tenant. (Default value)